rpm/six/Makefile

```diff
@@ -0,0 +1,43 @@
+NAME          = six
```

Comment:
Can you remind me why are we building six? This is probably a dependency in one of our AM components but that's managed via pip in the virtualenvs. Is this used by someone else?